binr/rax2/rax2.c

```diff
@@ -188,7 +191,9 @@ static int rax(char *str, int len, int last) {
 		}
 	}
 dotherax:
-
+	if (flags & (1 << 19)) { // -s
```

Comment:
also, it looks to me, that only few excpetions we need to remove that newline, but this shouldnt mess the entire code with a global newline variable. so this doesnt looks good to me